### PR TITLE
 aligning README.md and CONTRIBUTING.md to FINOS project blueprint languages

### PR DESCRIPTION
**Pull request by nikoganev**
aligning README.md and CONTRIBUTING.md to FINOS project blueprint languages

### CODE REVIEW

**Comment by maoo** [comment]
LGTM, build failed for other reasons, so merging the PR anyway

File: README.md [hunk 1]
```diff
@@ -44,10 +44,19 @@ The bot will provide you with help by asking it for `help`:
...
 
-Copyright 2017 Fintech Open Source Foundation
+## License
 
-Distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+The code in this repository is distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
 
 SPDX-License-Identifier: [Apache-2.0](https://spdx.org/licenses/Apache-2.0)
+
+Copyright 2017-2019 Fintech Open Source Foundation
```
Comment (pmonks) [inline README.md]: This copyright notice is incorrect.  [Copyright notices only include the year of first publication](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies) - they are inherently open ended from that year forward.

Comment (copiesofcopies) [reply README.md]: The notice is not incorrect. The Copyright Office's guidance says that a notice should include the year of first publication, not that it should be exclusive of other dates. For materials that are updated regularly over a period of time, it's common and often recommended to include a range of dates.

Comment (pmonks) [reply README.md]: @copiesofcopies [the Copyright Office disagrees...](https://www.copyright.gov/circs/circ03.pdf)

Comment (pmonks) [reply README.md]: @copiesofcopies sure - it doesn't say that Copyright Notices shouldn't include War and Peace either, but I think that not adding spurious elements that are left undefined by the relevant authority is common sense.

**Pull request merged**